README.md

```diff
@@ -202,6 +202,6 @@ Thimble uses the [throng](https://www.npmjs.com/package/throng) module to levera
 Contact Us
 -----------
 
-We're a friendly group, so feel free to chat with us in the "Thimble" channel on Mozilla Chat running on [Mattermost](https://about.mattermost.com). To access Mozilla Chat head to [this link]( http://chat.mozillafoundation.org). Note that you will need to create an account first.
+We're a friendly group, so feel free to chat with us in the "Thimble" channel on Mozilla Chat running on [Mattermost](https://about.mattermost.com). To access Mozilla Chat head to [this link]( http://chat.mozillafoundation.org). Note that you will need to create an account first. If you have signed in already, head [here](https://chat.mozillafoundation.org/mozilla/channels/thimble) to directly visit the Thimble channel.
```

Comment:
I think saying more is good in this case, since we're trying to make it as easy as possible for people who are new to find and get connected with us.